<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QWidget" name="SettingsWindow">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="OpenBookFileButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Open book file</string>
   </property>
  </widget>
  <widget class="QLabel" name="SelectedBookTextLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>98</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Selected Book File:</string>
   </property>
  </widget>
  <widget class="QLabel" name="BookFileName">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>531</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
    </property>
    <item>
     <widget class="QCheckBox" name="ignoreCaseCheckBox">
      <property name="text">
       <string>ignore case</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="ignorePunctuationCheckBox">
      <property name="text">
       <string>ignore punctuation</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="clearProgressButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>80</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>clear current progress</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
